app/controllers/vocab_sheets_controller.rb

```diff
@@ -12,7 +12,7 @@ def show
   end
 
   def update
-    @sheet.name = params[:vocab_sheet][:name]
+    @sheet.update_attributes!(vocab_sheet_params)
```

Comment:
I right now think it will never reach line 16 if the validations fail - the code will throw an exception on line 15

We could instead do `if @sheet.update(vocab_sheet_params)`

Reply:
Like this?